.buildkite/pipeline.yml

```diff
@@ -154,6 +154,13 @@ steps:
         agents:
           slurm_mem: 20GB
 
+      - label: "Slabplanet: albedo from temporal map"
+        key: "slabplanet_albedo_maptemporal"
+        command: "julia --color=yes --project=experiments/AMIP/modular/ experiments/AMIP/modular/coupler_driver_modular.jl --run_name slabplanet_albedo_maptemporal --FLOAT_TYPE Float64 --coupled true --surface_setup PrescribedSurface  --moist equil --vert_diff true --rad gray --energy_check true --mode_name slabplanet --t_end 10days --dt_save_to_sol 9days --dt_cpl 200 --dt 200secs --mono_surface true --h_elem 4 --precip_model 0M --anim true --albedo_type map_temporal --job_id slabplanet_albedo_maptemporal"
```

Comment:
@LenkaNovak (starting a thread so we can add comments) - I realized I left 2 `@show` statements in the ClimaLSM code that updates the albedo at each timestep ([here](https://github.com/CliMA/ClimaLSM.jl/blob/main/src/standalone/Bucket/Bucket.jl#L640C1-L641)). I think this is what's causing this run to exceed the log limit. I can fix it in ClimaLSM and make another patch release.

Reply:
Should I remove the `@show`s before merging this PR, or do you think it's okay to merge this as is for now and fix ClimaLSM later?

Reply:
I would personally remove the first, bump a patch release and update this PR again.

Reply:
Okay, I removed them in ClimaLSM, released ClimaLSM v0.3.2, and updated this PR to use that version. It should be fixed now